### PR TITLE
Bug 1615540 - Remove AWFY from Perfherder

### PR DESCRIPTION
**Pull request by kimberlythegeek**

### CODE REVIEW

**Comment by sarah-clements** [comment]
You'll also need to filter the frameworks by `enabled` in the UI.

**Comment by kimberlythegeek** [comment]
@sarah-clements ah, yes, forgot that step 👌

File: treeherder/perf/fixtures/performance_framework.json [hunk 1]
```diff
@@ -28,7 +28,7 @@
     "model": "perf.PerformanceFramework",
     "fields": {
       "name": "awfy",
-      "enabled": true
+      "enabled": false
```
Comment (ionutgoldan) [inline treeherder/perf/fixtures/performance_framework.json:31]: This will do. Thanks!

Comment (ionutgoldan) [inline treeherder/perf/fixtures/performance_framework.json:31]: This is the gist of this whole PR.

Still, the frontend needs to know which frameworks are enabled/disabled. That can be tweaked [here](https://github.com/mozilla/treeherder/blob/368c112266f4f276251a4886a5337fcb17b3a1e9/treeherder/webapp/api/performance_serializers.py#L54).

File: treeherder/webapp/api/performance_serializers.py [hunk 2]
```diff
@@ -51,7 +51,7 @@ class Meta:
 class PerformanceFrameworkSerializer(serializers.ModelSerializer):
     class Meta:
         model = PerformanceFramework
-        fields = ['id', 'name']
+        fields = ['id', 'name', 'enabled']
```
Comment (ionutgoldan) [inline treeherder/webapp/api/performance_serializers.py:54]: If we filter out the disabled tests, then this extra detail is redundant.

File: treeherder/webapp/api/performance_data.py [hunk 3]
```diff
@@ -193,7 +193,7 @@ def list(self, request, project):
 
 
 class PerformanceFrameworkViewSet(viewsets.ReadOnlyModelViewSet):
-    queryset = PerformanceFramework.objects.all()
+    queryset = PerformanceFramework.objects.all().filter(enabled=True)
```
Comment (ionutgoldan) [inline treeherder/webapp/api/performance_data.py:196]: How cool! Haven't really thought about this approach.
But I like it.

I think you can remove the `all()`, as it's redundant.

Comment (kimberlythegeek) [reply treeherder/webapp/api/performance_data.py:196]: Thanks!

**Comment by sarah-clements** [comment]
@kimberlythegeek Are you able to merge this? If not, I can do it.

**Comment by kimberlythegeek** [comment]
@sarah-clements I am not authorized, no

**Pull request merged**